spot_driver/spot_driver/spot_ros2.py

```diff
@@ -814,9 +815,9 @@ def handle_list_graph(self, request, response):
         """ROS service handler for listing graph_nav waypoint_ids"""
         try:
             self.node.get_logger().error(f'handle_list_graph: {request}')
-            self.spot_wrapper._clear_graph()
-            self.spot_wrapper._upload_graph_and_snapshots(request.upload_filepath)
-            response.waypoint_ids = self.spot_wrapper.list_graph(request.upload_filepath)
+            self.spot_wrapper.spot_graph_nav._clear_graph()
+            self.spot_wrapper.spot_graph_nav._upload_graph_and_snapshots(request.upload_filepath)
```

Comment:
Is there a reason these are private functions in spot_wrapper if they are going to be used here?